cli/lib/kontena/cli/services/services_helper.rb

```diff
@@ -287,41 +287,45 @@ def deploy_service(token, service_id, data)
 
         # @param [Hash] deployment
         # @return [String] multi-line
-        def render_service_deploy_instances(deployment)
+        def render_service_deploy_instances(deployment, vocabulary)
```

Comment:
A) How about just making it more robotic/generic:

```
⊛ ONGOING abcd/xyz-1234 on node foofy-bar-4567
```

B) How about

def state_inflections
  @state_inflections ||= {
     'ongoing' => 'Deploying',
     'default' => 'Deploy'
   }
end

def state_inflection(state)
  state_inflections.fetch(state) { state_inflections[:default] }
end
```

and then override the verb list method in the class that includes service helper.

or

C) something even better you come up with

Reply:
I'll just revert back to `Deploying ...` - it's technically correct, because it's a deployment. The spinner still says "Starting...".